draft-ietf-quic-transport.md

```diff
@@ -3843,11 +3843,8 @@ instead of dropping it.  Endpoints react to congestion by reducing their sending
 rate in response, as described in {{QUIC-RECOVERY}}.
 
 To use ECN, QUIC endpoints first determine whether a path supports ECN marking
-and the peer is able to access the ECN codepoint in the IP header.  A network
-path does not support ECN if ECN marked packets get dropped or ECN markings are
-rewritten on the path. An endpoint validates the use of ECN on the path, both
-during connection establishment and when migrating to a new path
-({{migration}}).
+and the peer is able to access the ECN codepoint in the IP header; see
+{{ecn-validation}}.
```

Comment:
I think the new text is wrong. Support by the endpoint isn't sufficient. The path validation is required to use ECN, and this spec needs to say this for each new path.

Reply:
I'm also in favour or keep this text here. It's a bit of redundant but I think it's useful to make this clear from the beginning.